.github/workflows/mkosi.yml

```diff
@@ -112,6 +112,18 @@ jobs:
         Format=squashfs
         EOF
 
+        # The emergency shell is not useful in the CI, as it just blocks for a long time before the job
+        # eventually times out. Override it to just shutdown immediately.
+        mkdir -p mkosi.presets/20-final/mkosi.extra/usr/lib/systemd/system/emergency.service.d/
+        tee mkosi.presets/20-final/mkosi.extra/usr/lib/systemd/system/emergency.service.d/poweroff.conf <<- EOF
+        [Service]
+        ExecStartPre=
+        ExecStart=
+        ExecStart=false
+        [Unit]
+        FailureAction=exit
```

Comment:
My OCD wants the [Unit] section to come first

Reply:
that can be accommodated